api/src/models/users.js

```diff
@@ -0,0 +1,33 @@
+const connection = require('../db/connection')
+
+class Users {
```

Comment:
So far this model is only used to create new authenticated users, but we could reuse it in a few places.